ros2_ws/src/rcdt_franka/nodes/close_gripper_node.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: Alliander N. V.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import rclpy
+from rclpy.node import Node
+from rclpy.action import ActionClient
+from rclpy.executors import MultiThreadedExecutor
+from rclpy.callback_groups import MutuallyExclusiveCallbackGroup
+from rclpy import logging
+from franka_msgs.action import Grasp
+from std_srvs.srv import Trigger
+
+ros_logger = logging.get_logger(__name__)
+
+
+class CloseGripper(Node):
+    def __init__(self) -> None:
+        super().__init__("close_gripper")
+
+        cbg_service = MutuallyExclusiveCallbackGroup()
+        self.create_service(
+            Trigger, "close_gripper", self.callback, callback_group=cbg_service
+        )
+
+        cbg_client = MutuallyExclusiveCallbackGroup()
+        self.client = ActionClient(
+            self, Grasp, "/fr3_gripper/grasp", callback_group=cbg_client
+        )
+
+    def callback(
+        self, _request: Trigger.Request, response: Trigger.Response
+    ) -> Trigger.Response:
+        response.success = self.close_gripper()
+        return response
```

Comment:
Hier returnen we de response met alleen de success-attribute gevuld. Ik was net de service aan het aanroepen, en dan is het message-attribuut leeg. Het kan fijn zijn voor iemand die een service aanroept dat de message is gevuld met het bericht dat nu wel gelogd wordt in de terminal waar de service draait.